app/dashboard/views.py

```diff
@@ -4050,6 +4061,27 @@ def hackathon_registration(request):
         except Exception as e:
             logger.error('Error while adding to chat', e)
 
+        if poll:
+            poll = json.loads(poll)
+            set_questions = {}
+            for entry in poll:
+                try:
+                    question = Question.objects.get(id=int(entry['name']))
```

Comment:
I'm guessing that you wrapped this is a try/except to catch if these two querysets fail. Lets use `get_object_or_404` instead